src/_P013_HCSR04.ino

```diff
@@ -255,8 +260,13 @@ boolean Plugin_013(byte function, struct EventStruct *event, String& string)
               log += state;
               addLog(LOG_LEVEL_INFO,log);
               switchstate[event->TaskIndex] = state;
-              UserVar[event->BaseVarIndex] = state;
-              event->sensorType = SENSOR_TYPE_SWITCH;
+              if (operatingMode == OPMODE_STATE) {
+                UserVar[event->BaseVarIndex] = state;
+                event->sensorType = SENSOR_TYPE_SWITCH;
```

Comment:
+ValueCount

Reply:
??

Reply:
@uzi18 Can you explain what you meant with your comment?
I also don't really see what you're missing here.